README.md

```diff
@@ -26,42 +36,53 @@ The component that is generated from the web app is the event website. Several s
 
 #### Technologies used
 
-The webapp generator uses the following technologies -
- 1. HTML/CSS/Javascript based frontend
- 2. [SASS](http://sass-lang.com/) - SASS to write optimized CSS.
- 3. [Node.js](http://nodejs.org) - Javascript for the generator backend
- 4. [Express.js](http://expressjs.com) - Server framework
- 5. [Handelbars](http://handlebarsjs.com) - Template for rendering
- 6. [Socket.io](http://socket.io) - For handling multi-user client, with progress of upload/generation
- 6. [WebdriverJs](https://www.npmjs.com/package/selenium-webdriver) - Official Javascript Implementation of Selenium for testing generated event websites
+**The webapp generator uses the following technologies :-**
+
+ **1. HTML/CSS/Javascript based frontend**
+ 
+ **2. [SASS](http://sass-lang.com/)** - *SASS to write optimized CSS.*
+ 
+ **3. [Node.js](http://nodejs.org)** - *Javascript for the generator backend*
+ 
+ **4. [Express.js](http://expressjs.com)** - *Server framework*
+ 
+ **5. [Handelbars](http://handlebarsjs.com)** - *Template for rendering*
+ 
+ **6. [Socket.io](http://socket.io)** - *For handling multi-user client, with progress of upload/generation*
+ 
+ **7. [WebdriverJs](https://www.npmjs.com/package/selenium-webdriver)** - *Official Javascript Implementation of Selenium for testing generated event websites*
 
 #### Components of the Generator
 
-1. Webform
+**1. Webform**
+
+ > **The source of the webform can be found [here](/src/www).**
 
- The source of the webform can be found [here](/src/www). It consists of -
+**It consists of :-**
 
- - [index.html](/src/www/index.html) - The webform page
- - [form.js](/src/www/js/form.js) - The script that uploads the zips, and starts the generator process
+ - **[index.html](/src/www/index.html)** - *The webform page*
+ - **[form.js](/src/www/js/form.js)** - *The script that uploads the zips, and starts the generator process*
 
-2. Generator
+**2. Generator**
 
- The generator runs on a ExpressJS server, using this main [app script](/src/app.js).
+ > **The generator runs on a ExpressJS server, using this main [app script](/src/app.js).**
```

Comment:
Lets not make this bold, and remove `> `

Reply:
Done, and Done :blush: